cmd/agentflow/basic_scraper.flow

```diff
@@ -0,0 +1,39 @@
+metrics scraper "default"  {
+    targets = [
+        {
+            targets = [{ "__address__" = "localhost:9001"}]
+            labels = {}
+        }
+    ]
+    receivers = [metrics.remote_write.default.receiver, metrics.remote_write.rw2.receiver]
```

Comment:
receivers accepts an array of receivers. The naming of receivers feels overloaded and not specific so VERY open to opinion here.